### PR TITLE
Add NetBSD icon.

### PR DESCRIPTION
**Pull request by fraggerfox**
**Issue:** #7062 
**Alexa rank:** 89,285
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

-  The hex value was chosen from the original logo at https://www.netbsd.org/gallery/logos.html
- ~The logo was manually vectorized.~ The Encapsulated PostScript from the website was used.

image

### CODE REVIEW

**Comment by adamrusted** [comment]
Did you use the EPS version of the logo available on the source page? Without looking too deeply, I'd say that's probably our best bet for an accurate vector.

**Comment by fraggerfox** [comment]
> Did you use the EPS version of the logo available on the source page? Without looking too deeply, I'd say that's probably our best bet for an accurate vector.

You are correct, I missed out the EPS link, I have regenerated the svg from the EPS, thank you for pointing this out.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6566,6 +6566,12 @@
             "source": "http://www.netapp.com/",
             "guidelines": "https://www.netapp.com/company/legal/trademark-guidelines/"
         },
+        {
+            "title": "NetBSD",
+            "hex": "FF6600",
+            "source": "https://www.netbsd.org/gallery/logos.html",
```
Comment (dirien) [inline _data/simple-icons.json:6572]: Can you write the source just with "https://www.netbsd.org" please.

**Review by dirien — CHANGES_REQUESTED** [review]
hi @fraggerfox,

Perfect fit!

image

Left a comment on the source tag. 

After this, we can merge this PR

File: _data/simple-icons.json (hunk 1 above)
Comment (fraggerfox) [reply _data/simple-icons.json]: Should be done in the latest commit.

**Comment by dirien** [comment]
thx @fraggerfox! Merged! :)

**Comment by fraggerfox** [comment]
@dirien could you have a look at the related PR #7061 (which is the package manager for NetBSD)? (When you get time of course), Thank you.

**Pull request merged**